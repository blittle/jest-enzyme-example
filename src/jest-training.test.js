import React from 'react';
import MyComponent from './jest-training.js';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';

describe('jest training', function() {
	// it('should render', function() {
	// 	const component = renderer.create(
	// 		<MyComponent />
	// 	);
	// 	let tree = component.toJSON();
	// 	expect(tree).toMatchSnapshot();
	// });
  //
	// it('should click', function() {
	// 	const spy = jasmine.createSpy();
  //
	// 	const wrapper = shallow(<MyComponent clicked={spy} />);
	// 	wrapper.find('div').simulate('click');
  //
	// 	expect(spy).toHaveBeenCalled();
	// });
});
