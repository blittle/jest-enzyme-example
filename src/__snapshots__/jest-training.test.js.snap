exports[`jest training should click 1`] = `
<div
  className="wrapper"
  onClick={[Function]}>
  Hello It changed
</div>
`;

exports[`jest training should render 1`] = `
<div
  className="wrapper"
  onClick={undefined}>
  Hello It changed
</div>
`;
